spec/models/comment_spec.rb

```diff
@@ -29,4 +29,9 @@
     subject.post = nil
     expect(subject).to_not be_valid
   end
+
+  it 'Comment should not be valid withou text' do
+    subject.text = nil
+    expect(subject).to_not be_valid
```

Comment:
- [Optional] Dear @Trast00, Please notice your validation is validating String type `validates :text, presence: true` but your test is testing against `nil` which is from a `NillClass``. This is unexpected behavior. Please test with an empty string instead. And please remember `nill` is not the same as `""`. Nill can be evaluated as false but empty string is always true in Ruby. 
![Screenshot from 2023-04-06 16-42-00](https://user-images.githubusercontent.com/69077061/230396727-d1e026cf-6b06-4c4c-8cb6-0271b2721867.png)

Reply:
Thank you, for that notice. I will definitely try that! 👍